packages/core/src/connectors/aliyun-dm/index.ts

```diff
@@ -0,0 +1,12 @@
+import { ConnectorType } from '@logto/schemas';
+
+import { ConnectorMetadata } from '../types';
+
+export const metadata: ConnectorMetadata = {
+  id: 'aliyun-dm',
+  type: ConnectorType.SMS,
+  name: '阿里云邮件推送',
+  logo: './logo.png',
```

Comment:
should be an URL?

Reply:
thought a while, will be better to store an URL or relative URL or some Logto-defined schema for internal reference. e.g. `logto-cdn://connectors/aligyun-dm/logo` or use something like`from-cdn` as a special placeholder

Reply:
We can auto detect the URL type:
1. Starts with `http` or `https`, then is a third-party CDN url.
2. Otherwise, is a relative URL, we can replace is with our cdn configurate if needed.

Reply:
Ok with it. Should we use some special protocol (e.g. `logto-static://`) to explicitly indicate our internal cdn / static url? since I was trying to find the file inside the repo when I saw the relative path here

Reply:
LOG-1301 this will be handled in query for API.